amit/index.js

```diff
@@ -0,0 +1,5 @@
+// 1
```

Comment:
Here is the solutions!
// 1
// NaN stands for Not-a-Number.
console.log(isNaN("hello"));
console.log(isNaN(3));
console.log(typeof NaN);
// Type of NaN is a number

// 2
console.log(Infinity / 0);
console.log(Infinity / Infinity);
console.log(1 / 0);

Reply:
console.log(typeof NaN);
// Type of NaN is a number
I don'n understand these lines

Reply:
Well NaN(Not a Number) is a property that checks! This property indicates that a value is not a legal number!
So using typeof we can find out which type of data we have. Data Type of NaN is actually Number!:)
https://javascriptrefined.io/nan-and-typeof-36cd6e2a4e43